app/assets/javascripts/models/beta/history.js

```diff
@@ -0,0 +1,38 @@
+import { sortBy, has } from 'underscore';
+import { findById } from './user';
+
+export const getDate = (oldValue, newValue) =>
+  [
+    oldValue ? I18n.l("date.formats.long", oldValue) : null,
+    oldValue ? I18n.l("date.formats.long", newValue) : null
+  ];
+
+const isDate = key => key.endsWith("_at");
+
+const isId = key => key.endsWith("_id");
+
+const isInitials = key => key === 'owned_by_initials';
+
+export const deserializeChanges = changes =>
+  Object.keys(changes).map(key => {
```

Comment:
I believe you can you [mapObject](https://underscorejs.org/#mapObject) here.

Reply:
Nice. Thanks!